starter-code/js/index.js

```diff
@@ -1,20 +1,48 @@
+var product = {};
+
 function deleteItem(e){
 
 }
 
-function getPriceByProduct(itemNode){
-
+// function getPriceByProduct(itemNode){
+function getPriceByProduct(){
+  var price = document.createElement('span');
+  var priceText = document.createTextNode(product.price + '$');
+  price.appendChild(priceText);
+  document.getElementById('wrapper').appendChild(price);
 }
 
 function updatePriceByProduct(productPrice, index){
-
+  productPrice = product.price * index;
+  product.quantity = document.getElementById("input").value;
+  var productUpdatedPrice = document.createElement('span');
+  var UpdatedPriceText = document.createTextNode(productPrice + '$');
+  productUpdatedPrice.appendChild(UpdatedPriceText);
+  document.getElementById('wrapper').appendChild(productUpdatedPrice);
 }
 
-function getTotalPrice() {
-
+function getTotalPrice(productPrice, index) {
+  product.quantity = document.getElementById("input").value;                       //value of the input
+  console.log(product.quantity);
+  index = product.quantity;
+  productPrice = product.price * index;
+  updatePriceByProduct(product.price, product.quantity);
+  var productUpdatedPrice = document.createElement('span');
+  var UpdatedPriceText = document.createTextNode(productPrice + '$');
+  productUpdatedPrice.appendChild(UpdatedPriceText);
+  document.getElementById('footer').appendChild(productUpdatedPrice);
```

Comment:
Doing this way, it will append a `span` element each time you click the button.
We could just take the span element and update his textContent.